test/test_misc.py

```diff
@@ -2378,6 +2378,25 @@ def count_graph_break_msgs(msgs):
             g1(torch.randn(10), torch.randn(10))
             self.assertEqual(count_graph_break_msgs(log.output), 1)
 
+    def test_inplace_param_update(self):
+        def fn(param, y):
+            prev_grad = torch.is_grad_enabled()
+            try:
+                torch.set_grad_enabled(False)
```

Comment:
Can we test setting the same value twice?

Reply:
Added